lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,117 @@
+
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+  const renderEl = rtlRenderElement(NxPageHeader, {});
+
+ it('renders a top-level header element', function() {
+    renderEl();
+    const header = screen.getByRole('banner');
+    expect(header).toBeInTheDocument();
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders the default logo within the header', function() {
+    renderEl();
+    const logo = screen.getByRole('img');
+    expect(logo).toBeInTheDocument();
+  });
+
+  it('renders a custom logo instead of the default one when given logo prop', function() {
+    renderEl({ logo: { path: 'foo', alt: 'bar' } });
+    const logo = screen.getByRole('img');
+
+    expect(logo).toHaveAttribute('src', 'foo');
+    expect(logo).toHaveAttribute('alt', 'bar');
+    // ensures there is only one logo
+    expect(screen.getAllByRole('img').length).toEqual(1);
+  });
+
+  it('renders logo link when given homeLink prop', function() {
+    renderEl({ homeLink: '#home'});    
+    const homeLink = screen.getByRole('link', {name: 'Home'}),
+      homeLinkLogo = screen.getByRole('img');
 
-    expect(component).toHaveProp('links', props.links);
-    expect(component).toHaveProp('homeLink', props.homeLink);
-    expect(component.children()).toMatchElement(props.children);
+    expect(homeLink).toBeInTheDocument();
+    expect(homeLink).toHaveAttribute('href','#home');
+    expect(homeLinkLogo).toBeInTheDocument();
   });
 
-  it('passes a logo with a webpack-generated path and proper alt tag', function() {
-    const logo = shallow(getShallowComponent().prop('logo'));
+  it('renders links in header when given links prop', function() {
+    renderEl({links: [{ name: 'foo', href: '#bar' }, { name: 'baz', href: '#qux', current: true }]});
+    const link1 = screen.getByRole('link', {name: 'foo'}),
+      link2 = screen.getByRole('link', {name: 'baz'})
 
-    // this path comes from imgMock.ts
-    expect(logo).toHaveDisplayName('img');
-    expect(logo).toHaveProp('src', 'path/to/asset.png');
-    expect(logo).toHaveProp('alt', 'Sonatype');
-    expect(logo).toHaveClassName('nx-product__logo-image');
+    expect(link1).toBeInTheDocument();
+    expect(link1).toHaveAttribute('href', '#bar');
+    expect(link2).toBeInTheDocument();
+    expect(link2).toHaveAttribute('href', '#qux');
+    expect(link2).toHaveAttribute('aria-current', 'page');
```

Comment:
> Should assert that the other link does not have `aria-current="page"`. Personally I'd split out the `aria-current` checks into a separate test

I created a separate test here: [72fb68c](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/72fb68cbc86215765364ed7a4725ec094ebb2342)

The non-current links are not supposed to have the `aria-current` attribute at all, so I went with `expect(link).not.toHaveAttribute('aria-current')` 
instead of 
`expect(link).toHaveAttribute('aria-current', 'page');`

If you prefer the latter, let me know